src/main/resources/js/view.issue.module.js

```diff
@@ -1,5 +1,13 @@
 function fillIssueModule() {
 	console.log("view.issue.module fillIssueModule");
+	
+	var exportMenuItem = document.getElementById("export-as-table-link");
+	exportMenuItem.addEventListener("click", function(e) {
+	    e.preventDefault();
+	    e.stopPropagation();
```

Comment:
Does anybody know why the user is propagated to login page?

Reply:
No clue, on some pages it worked like that, at least at one it didn´t. The only way which worked always was:
AJS.$(document).off().on("click", "#export-as-table-link", function(e) {
to add multiple listeners here you need to comma seperate the buttons, but then also differentiate between them afterwards I guess:
AJS.$(document).off().on("click", "#export-as-table-link, #anotherButtonId", function(e) {